src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -121,6 +159,13 @@ namespace Microsoft.FSharp.Collections
         ///
         /// <returns>The result sequence.</returns>
         ///
+        /// <example>
+        /// <code lang="fsharp">
+        ///     ([1; 2; 3] :> IEnumerable) |> Seq.cast{int} // evaluates to seq [1; 2; 3], explicitly typed as int seq
```

Comment:
Does `{int}` format correctly?  I think you need `&lt;int&gt;`?

Reply:
Unless `<code>` has special rules, I'm not sure, I doubt it

Reply:
Re-reading the docs, it's for cref, there you use {} for generics.